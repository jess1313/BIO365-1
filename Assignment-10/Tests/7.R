suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("7.R"))))

factor1 = factor(c(rep("cold", 10), rep("warm", 27), rep("hot", 19)), levels=c("cold", "warm", "hot"))
factor2 = factor(c(1:10, 1:5, 1:7, 4:9, 3:8))

print(round(calcProportions(factor1), 8))
print(round(calcProportions(factor2), 8))