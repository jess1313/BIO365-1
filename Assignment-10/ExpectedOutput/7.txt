     cold      warm       hot 
0.1785714 0.4821429 0.3392857 
         1          2          3          4          5          6          7 
0.08823529 0.08823529 0.11764706 0.14705882 0.14705882 0.11764706 0.11764706 
         8          9         10 
0.08823529 0.05882353 0.02941176 
